sdh-proxy.c

```diff
@@ -65,9 +65,18 @@ int do_exit = 0;
 // Toggled with -d on command line. Displays debug info. 
 int debug = 0;
 
+// Toggled with -l on command line. Write Stats to stat.log
+int logstat = 0;
+char logstatfile[] = "sdh.stat";
+
 // Does nothing at the moment. Will enable rewriting subnet info if relevant. 
 int do_network_rewrite = 0; // Rewrite IP broadcast address for the new
                             // network interface
+int32_t pkt_rx = 0;
+int32_t pkt_tx = 0;
+int32_t pkt_drop = 0;
+unsigned short int pkt_stats[65535];
```

Comment:
Hey, indeed on the event the counter overflow several times, especially  27015 :) 
For my purpose it was ok because I just tracked difference between previeous poll in a 10s resolution. But could be changed to int64